it/uk/gov/hmrc/cipphonenumberverification/VerificationIntegrationSpec.scala

```diff
@@ -45,5 +45,21 @@ class VerificationIntegrationSpec extends AnyWordSpec
 
       response.status shouldBe 202
     }
+
+    "return 200 with valid fixed line number" in {
```

Comment:
we don't need to add IT tests for each and every scenario, we just need one example of each happy path and unhappy path